bioio_ome_zarr/tests/test_s3_read.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+from bioio_ome_zarr import Reader
+
+
+def test_ome_zarr_reader() -> None:
+    # ARRANGE
+    uri = (
+        "s3://allencell/aics/nuc_morph_data"
+        "/data_for_analysis/baseline_colonies/20200323_09_small/raw.ome.zarr"
```

Comment:
precedent being set: permanently situated public data that we run tests on!  (This could skew access stats if running the test becomes an outlier compared to actual people wanting that image)